src/utils/viewer/viewer.cpp

```diff
@@ -525,6 +549,106 @@ void Viewer::switchCameraType() {
   }
 }
 
+void Viewer::saveCameraTransformToFile() {
+  if (!Cr::Utility::Directory::exists(saved_transformations_directory)) {
+    Cr::Utility::Directory::mkpath(saved_transformations_directory);
+  }
+
+  // update temporary save
+  lastCameraSave_ = renderCamera_->node().absoluteTransformation();
+  cameraSaved = true;
+
+  // update save in file system
+  saveNodeTransformToFile(renderCamera_->node(),
+                          saved_transformations_directory + "camera." +
```

Comment:
you can use:
 [Utility::format()](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility.html#a5b85466788fc3d71b143909dda1ef44a),